packages/dd-trace/src/tracer.js

```diff
@@ -16,7 +16,7 @@ class DatadogTracer extends Tracer {
   constructor (config) {
     super(config)
 
-    const Scope = getScope()
+    const Scope = getScope(config.scope)
```

Comment:
Is there a test covering this? It seems like this should make or break a test

Reply:
Actually, this should have made the integration tests fail, not sure why this wasn't the case, I'll look into it.

Reply:
Ah, it's because they are not merged yet: https://github.com/DataDog/dd-trace-js/pull/1224

Reply:
Ah yeah, in that case, disregard.